buildroot/packages/configd/configd.h

```diff
@@ -0,0 +1,11 @@
+#ifndef configd
+#define configd
+
+#define CONFIG_FILE "/etc/switch.json"
```

Comment:
It would be preferable if this was only taken as the default, and you allowed for the user to override this at runtime with an argument (e.g. `-c /path/to/config.json`)

Reply:
I'd assumed at some point this package would be configured at boot and therefore would get little to no input from the user. But, even if that is the case, that doesn't mean it wouldn't be useful to set, e.g., in the init file :+1: just pushed a commit to implement that arg.

Reply:
Ah . . . allowing an arbitrary path to this file might present an issue with other tools like the web interface which currently just [`cat`s the file in a cgi script](https://github.com/hall/postmerkos-ui/blob/330bd672ed2ffee95c919134cf7d4b1506e5fe8e/assets/cgi-bin/config#L5).

Not sure yet if there's a reasonable alternative there. I suppose `ps aux` returns the full command (with args and process id) then `pwd /proc/${id}/cwd` is the directory the process was started from. So that gives _some_ way of recovering the absolute path.